nautobot_ssot/views.py

```diff
@@ -134,12 +132,6 @@ class SyncView(ObjectView):
     queryset = Sync.annotated_queryset()
     template_name = "nautobot_ssot/sync_detail.html"
 
-    def get_extra_context(self, request, instance):
```

Comment:
what do we replace this for?

Reply:
This is not being used by any of the templates